### DISCUSSION TITLE
E1 Outdoor - Link to v1584_22120966

### DISCUSSION THREAD

**Discussion by buggyglint** [AT0myks/reolink-fw-archive#11] [Firmware sharing]
Here is another firmware for the E1 Outdoor, '1584_22120966'

[https://support-d.reolink.com/attachments/token/rngyK0yBtNIqpdAEaAuUDtfVr/?name=IPC_523SD8.1584_22120966.E1-Outdoor.OV05A10.5MP.WIFI1021.PTZ.REOLINK.pak](url)

Thanks for the many "hidden" firmwares!

**Comment by AT0myks** [comment]
Thank you for sharing!
